src/common/components/clusterWizard/clusterDetailsValidation.ts

```diff
@@ -74,7 +72,7 @@ export const getClusterDetailsInitialValues = ({
     diskEncryptionMode: cluster?.diskEncryption?.mode ?? 'tpmv2',
     diskEncryptionTangServers: parseTangServers(cluster?.diskEncryption?.tangServers),
     diskEncryption: cluster?.diskEncryption ?? {},
-    cpuArchitecture: cluster?.cpuArchitecture || CpuArchitecture.x86,
+    cpuArchitecture: cluster?.cpuArchitecture || getDefaultCpuArchitecture(),
```

Comment:
Instead of a hardcoded default cpuArchitecture, we now have a place where we can configure it so that everywhere will use that default.
This needs to be implemented later, for now it's just about preparing the code for that change.